src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx

```diff
@@ -44,159 +34,168 @@ type State = {
-            >
-              <Confirm
-                message={t('Are you sure you want to remove this rule?')}
-                onConfirm={this.onDelete}
-                disabled={!canEdit}
-              >
-                <Button size="xsmall" icon="icon-trash" />
-              </Confirm>
-            </Tooltip>
+            {data.actions.map((action, i) => (
+              <div key={i}>{action.name}</div>
+            ))}
           </Actions>
-        </PanelHeader>
-
-        <PanelBody>
-          <RuleDescriptionRow>
-            <RuleDescriptionColumn>
-              {data.conditions.length !== 0 && (
-                <Condition>
-                  <h6>
-                    When <strong>{data.actionMatch}</strong> of these conditions are met:
-                  </h6>
-                  <table className="conditions-list table">
-                    <tbody>
-                      {data.conditions.map((condition, i) => (
-                        <tr key={i}>
-                          <td>{condition.name}</td>
-                        </tr>
-                      ))}
-                    </tbody>
-                  </table>
-                </Condition>
-              )}
-            </RuleDescriptionColumn>
-            <RuleDescriptionColumn>
-              {data.actions.length !== 0 && (
-                <Condition>
-                  <h6>
-                    Take these actions at most{' '}
-                    <strong>
-                      once every <Duration seconds={data.frequency * 60} />
-                    </strong>{' '}
-                    for an issue:
-                  </h6>
-                  <table className="actions-list table">
-                    <tbody>
-                      {data.actions.map((action, i) => (
-                        <tr key={i}>
-                          <td>{action.name}</td>
-                        </tr>
-                      ))}
-                    </tbody>
-                  </table>
-                </Condition>
-              )}
-            </RuleDescriptionColumn>
-          </RuleDescriptionRow>
-        </PanelBody>
-      </Panel>
+        </TriggerAndActions>
+      </RuleItem>
     );
   }
+
+  renderMetricRule(data: SavedIncidentRule) {
+    const {params, routes, location, canEdit} = this.props;
+    const editLink = recreateRoute(`metric-rules/${data.id}/`, {
+      params,
+      routes,
+      location,
+    });
+
+    return (
+      <RuleItem>
+        <RuleType>{t('Metric')}</RuleType>
+        <div>
+          {canEdit ? <RuleName to={editLink}>{data.name}</RuleName> : data.name}
+          <RuleDescription />
+        </div>
+
+        <div>
+          {data.triggers.length !== 0 &&
+            data.triggers.map((trigger, i) => (
+              <TriggerAndActions key={i}>
+                <Trigger>
+                  <StatusBadge>{trigger.label}</StatusBadge>
+                  <div>
+                    {data.aggregations[0] === 0 ? t('Events') : t('Users')}{' '}
```

Comment:
wouldn't be better to add a css space here instead of `{'  '}` ?

Reply:
If we used a CSS space, we'd have to guess at a pixel size, I would rather have a more consistent whitespace by using the space character here.

Reply:
I was thinking about using template strings here, but I like having the explicit space here since it's less likely to be accidentally removed in the future.